.github/workflows/release.yml

```diff
@@ -83,3 +83,41 @@ jobs:
           tags: ${{ env.DOCKER_HUB_TAG }}
           cache-from: type=registry,ref=${{ env.DOCKER_HUB_TAG }}
           cache-to: type=inline
+
+  jmeter:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_HUB_REPOSITORY: maestro-jmeter
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+
+      - uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/v}
+
+      - name: Set Env Variables
+        run: |
+          echo "DOCKER_HUB_VERSION=${{ steps.get_version.outputs.VERSION }}" >> $GITHUB_ENV
```

Comment:
I think, that this might be a good thing to have our version only pinned here without any Jmeter version mentioned. 

Because if start having something like `5.3.1_0.3` it will break semantic versioning